R/rowClust.R

```diff
@@ -0,0 +1,83 @@
+#' Clusters rowData using \code{fastcluster::hclust}
+#'
+#' \code{rowClust} Performs hierarchical clustering on rowData of the 
+#' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}, 
+#' by using the dissimilarity function provided at \code{dissimilarity_FUN}.
+#'
+#' @param x a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object.
+#'   
+#' @param abund_values a \code{character} vector, name of the assay containing the data.
+#' 
+#' @param name a \code{character} vector, name of the new column at \code{rowData} where to place cluster ids.
+#' 
+#' @param clustering_method a \code{character} vector representing the clustering 
+#' method to be used, same as methods available at \code{\link[hclust]{hclust}}.
+#' 
+#' @param transposed \code{logical} whether the data should be transposed.
+#' 
+#' @param dissimilarity_FUN \code{stats::cor} function to compute the dissimilarity matrix
+#' 
+#' @param ... other arguments passed onto \code{\link[cutree]{cutree}}
+#'
+#' @return \code{x} with additional \code{\link{rowData}} named \code{*name*}
+#'
+#' @export
```

Comment:
I think we typically do not use spaces between these lines. Check other functions, ensure uniform design,

Reply:
I have checked existing functions and it seems there is a space between each description (or when using @)?

Reply:
Right, so it  is. Then all is good.